editor/src/components/navigator/navigator-item/component-picker-context-menu.tsx

```diff
@@ -127,12 +144,12 @@ export function preferredChildrenForTarget(
 
   // TODO: we don't deal with components registered with the same name in multiple files
   if (registeredComponent != null) {
-    if (insertionTarget === 'insert-as-child' || insertionTarget === 'replace-target') {
+    if (isChildInsertionTarget(insertionTarget) || isReplaceInsertionTarget('replace-target')) {
```

Comment:
I know that this is soon going to be replaced after merging with my PR, but please can you double check that this tautology is gone!
```
isReplaceInsertionTarget('replace-target')
```